app/main.development.js

```diff
@@ -435,6 +435,14 @@ function createDexWindow(serverAddress) {
   dexWindow.once("ready-to-show", () => {
     dexWindow.show();
   });
+  dexWindow.webContents.on("will-navigate", (e, url) => {
+    mainWindow.webContents.send("open-external", url);
+    e.preventDefault();
+  });
+  dexWindow.webContents.setWindowOpenHandler(({ url }) => {
+    mainWindow.webContents.send("open-external", url);
+    return { action: "deny" };
+  });
```

Comment:
@matheusd @bgptr These are the combined fixes from https://github.com/decred/decrediton/pull/3761 and comments.

They seem to work right.  Will rebase these out if https://github.com/decred/decrediton/pull/3761 goes in first, which would be good.

Reply:
I have removed the `"will-navigate"` code. Only the `setWindowOpenHandler` is needed to deal with the `_blank` targets, and `"will-navigate"` was catching the navigation on logout and opening another window incorrectly.